src/components/views/rooms/MessageComposer.tsx

```diff
@@ -602,6 +602,8 @@ export class MessageComposer extends React.Component<IProps, IState> {
                 className={classes}
                 ref={this.ref}
                 aria-describedby={this.state.recordingTimeLeftSeconds ? this.tooltipId : undefined}
+                aria-label="Message composer"
+                role="navigation"
```

Comment:
What's your reasoning for the "navigation" `role` here? "region" would probably be more appropriate, or even "form".

If the former, you'd be better off replacing the `<div>` element  with [`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section), and not including a `role` at all.

Reply:
Oh, I had forgotten about that. It was an attempt to get the region into the Voiceover rotor to no avail. I'll delete it again.